de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/JsonHandler.java

```diff
@@ -67,22 +77,53 @@ public void applyJoinRelation(FskPortObject fskObj, List<JoinRelationAdvanced> j
             String resourcePath = sourceModel.getGeneratedResourcesDirectory().get()
                 .getAbsolutePath().replaceAll("\\\\", "/") + "/";
             String jsonPath = resourcePath + JSON_FILE_NAME;
-            loadParametersIntoWorkspace(jsonPath, sourceParameter, param.getId());

```

Comment:
I cannot write on 75 since it did not change but there is a NPE there. SwaggerUtil.getParameter may return null and needs to be checked.